### PR TITLE
Remove purple coloring from DocumentBar and PostCard

### LINKED ISSUE #65168: Remove purple coloring from DocumentBar
As an affordance, the purple color applied in the DocumentBar is not helping to communicate the "global" nature of templates, synced patterns, and template parts. It's just not connective in this presentation, and with the addition of post type labels in https://github.com/WordPress/gutenberg/issues/65167 — arguably, not necessary. 

I propose we simplify a bit, removing the purple color within the DocumentBar. 

### Current 

current

### Proposed 

proposed

### PR DESCRIPTION
**Pull request by ntsekouras**
## What?
Resolves: https://github.com/WordPress/gutenberg/issues/65168

From the issue:

>As an affordance, the purple color applied in the DocumentBar is not helping to communicate the "global" nature of templates, synced patterns, and template parts. It's just not connective in this presentation, and with the addition of post type labels in https://github.com/WordPress/gutenberg/issues/65167 — arguably, not necessary.

> I propose we simplify a bit, removing the purple color within the DocumentBar and PostCard

## Testing Instructions
There should be no purple coloring in the above components for templates, synced patterns, and template parts.

## Screenshots

Check the `Save your changes.` command(s).

Before            |  After
:-------------------------:|:-------------------------:
Screenshot 2024-10-25 at 12 21 07 PM |after

### CODE REVIEW

**Review by jasmussen — APPROVED** [review]
Thank you!

**Comment by annezazu** [comment]
Noting a +1 from me here pulling from this prior issue: https://github.com/WordPress/gutenberg/issues/62039 Thanks for working on this.

File: packages/editor/src/components/post-card-panel/style.scss [hunk 1]
```diff
@@ -34,14 +34,6 @@
...
 }
 
-.editor-post-card-panel__icon.is-sync {
-	fill: var(--wp-block-synced-color);
-
-	& + .editor-post-card-panel__title {
-		color: var(--wp-block-synced-color);
-	}
-}
-
```
Comment (youknowriad) [inline packages/editor/src/components/post-card-panel/style.scss:44]: Should we remove is-sync and is-global from the component. That would be a nice cleanup.

Comment (ntsekouras) [reply packages/editor/src/components/post-card-panel/style.scss:44]: Oh, yes.. I'll create a follow up.

Comment (ntsekouras) [reply packages/editor/src/components/post-card-panel/style.scss:44]: https://github.com/WordPress/gutenberg/pull/66472

**Pull request merged**